github-merge.py

```diff
@@ -200,6 +200,8 @@ def print_merge_details(pull, title, branch, base_branch, head_branch, acks, mes
             print('{}Top commit has no ACKs!{}'.format(ATTR_WARN, ATTR_RESET))
     if message is not None and '@' in message:
         print('{}Merge message contains an @!{}'.format(ATTR_WARN, ATTR_RESET))
+    if message is not None and '<!-' in message:
+        print('{}Merge message contains an html comment!{}'.format(ATTR_WARN, ATTR_RESET))
```

Comment:
Looking at this more closely, this seems incorrect. `message` is overwritten by the `acks` loop above. Due to Python's lack of varilable scopring it contains the last ACK message.

Reply:
I saw that too, but I think this was tested for `@` (works the same)

And 18380 can be used to test the html comment

Reply:
Hmmm

```python
>>> def fun(i):
...  print(i)
...  for i in range(2):
...   print(i)
...  print(i)
... 
>>> fun(333)
333
0
1
1

Reply:
Oh, it works because it has no ACKs :man_facepalming: